styles.css

```diff
@@ -337,3 +327,1067 @@ body {
   flex-grow: 0;
   margin: 12px 0;
 }
+
+.header-shapes {
+  /* header-shapes mobile */
+
+  position: absolute;
+  height: 600px;
+  left: 0;
+  top: 104px;
+}
+
+.header-background {
+  /* header-background-mobile */
+
+  position: absolute;
+  width: 375px;
+  height: 718px;
+  left: 0;
+  top: 0;
+
+  /* Neutrals/N0 */
+
+  background: #fff;
+  visibility: hidden;
+}
+
+.works {
+  /* Works */
+
+  /* Auto layout */
+
+  display: grid;
+  align-items: center;
+  padding: 114px 24px;
+  position: absolute;
+  width: 327px;
+  height: 2560px;
+  left: 0;
+  top: 718px;
+}
+
+.card-works-1 {
+  /* Card Works */
+
+  /* Auto layout */
+
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 16px;
+  position: static;
+  width: 327px;
+  height: 575px;
+  left: 24px;
+  top: 114px;
```

Comment:
- [ ] You have `.card-works-1`, `.card-works-2`, `.card-works-3`, and `.card-works-4` but these classes have basically the same styles and you are repeating yourself. Kindly just create a `.card-works` class and use it on all your project cards to have a cleaner and more maintainable CSS code 😎